src/toil/test/wdl/builtinTest.py

```diff
@@ -58,6 +62,46 @@ def testFn_Floor(self):
         assert floor(1.999) == 1
         assert floor(-1.5) == -2
 
+    def testFn_ReadInt(self):
+        """Test the wdl built-in functional equivalent of 'read_int()'."""
+        num = 10
+        path = self._write_temp_file('read_int', content=str(num))
+        self.assertEqual(num, read_int(path))
+
+        num = 10.5
+        path = self._write_temp_file('read_int', content=str(num))
+        self.assertRaises(ValueError, read_int, path)
+
+    def testFn_ReadString(self):
+        """Test the wdl built-in functional equivalent of 'read_string()'."""
+        some_str = 'some string'
+        path = self._write_temp_file('read_string', content=some_str)
+        self.assertEqual(some_str, read_string(path))
+
+        # with new line characters
+        path = self._write_temp_file('read_string', content=some_str + '\n\n')
```

Comment:
The WDL spec says that Cromwell only cuts off trailing newlines:

```No trailing newline characters should be included```

But `read_string()` seems to `strip()` from the front and the end.  Is this the same as Cromwell's behavior?

Preceding newlines should be added as a test based on Cromwell's behavior.

Reply:
Good catch! I did not realize `strip()` removes preceding newlines.. Although, it seems like Cromwell removes preceding newlines as well. For reference, I am using `cromwell-52.jar` if that makes a difference at all. 

Either Cromwell isn't following their own spec, or, perhaps, "`[the file] is expected to contain 1 line with 1 string on it`" implies that there shouldn't be any preceding newlines to begin with?

I'll update the test as soon as possible. Thanks for the review!